app/javascript/application.js

```diff
@@ -1,3 +1,8 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import '@hotwired/turbo-rails'
 import 'controllers'
+
+// This is to help with breaking out of a turbo frame.
+Turbo.StreamActions.full_page_redirect = function () { // eslint-disable-line no-undef
```

Comment:
How do you feel about newer JS idioms like the fat arrow function syntax?

```suggestion
Turbo.StreamActions.full_page_redirect = () => { // eslint-disable-line no-undef
```

Reply:
Javascript is a mystery ...
<img width="550" alt="image" src="https://github.com/user-attachments/assets/bf026228-42fa-4c66-a687-ebb357643d89" />

Reply:
Sigh. Yeah, that makes no sense to me either.